client/src/components/ImageStrip.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { CarouselImageStrip } from './styles';
+
+const ImageStrip = ({ image, onImageClick }) => (
```

Comment:
If I remember correctly, this seemed to be more of a complex component than the code makes it seem here (I remember you scrolling through a list of images and having a bar at the bottom), perhaps that was after this code was written but if not, I'm not sure how you made all that work with such simple code here - nice job if so

Reply:
Yep, this is before I got that going. Thanks for the review Tyler!